libdnf/dnf-sack.cpp

```diff
@@ -402,6 +402,7 @@ load_ext(DnfSack *sack, HyRepo hrepo, _hy_repo_repodata which_repodata,
         done = TRUE;
     }
     if (error && *error) {
+        g_prefix_error(error, "write_ext(%d) failed to use %s: ", which_repodata, fn_cache);
```

Comment:
I don't like adding function names to the error messages. The messages end up being read by the end users for the ost part and they don't care about our function names. Better to provide a sensible description which should unambiguously point to this spot in the code anyway. (FWIW this has been completely rewritten in dnf 5; you can check out that code to see what I've done there and hopefully you'll find it much nicer. Not sure how much it'd help you with the error messages here but it might :slightly_smiling_face: )

Reply:
Ok, I have removed the function names from the added error messages.
(This reminds me I should probably port this to dnf5.)